### ISSUE TITLE
Unable to add monitored cluster with 1.5.0

### ISSUE THREAD

**Issue by dribblecastle** [codership/galera-manager-support#24]
I've deployed this successfully in our staging environment but was starting to install it in our production environment and I've run into the following problem.  Running 1.5.0. 

When I add a monitored cluster the UI hangs/spins on CREATING.  I've been patient (10+ minutes) and if I refresh the page the cluster appears listed but has a  Running Job of "cluster deploy".    If I try to deploy a node from it the "Check Access" button will hang in the same way.  

I've tried to force delete and recreate a new cluster. I've uninstalled and reinstalled Galera Manager via apt-get purge.   I'm currently running Ubuntu 20.04 with a 10.3.31 Galera cluster.    I don't see any output in terms of logs at /var/log/gmd.   The Logs UI shows "Log is empty" 

Any suggestions?  Thanks in advance.

**Comment by denisgcm** [comment]
Thank you for reaching out to us. Could you please post a screenshot of the step where it hangs?

**Comment by dribblecastle** [comment]
<img width="910" alt="image" src="https://user-images.githubusercontent.com/6880310/153663986-31f2b4b8-fb8d-4d75-9091-79e429bde6a1.png">

Sorry didn't realize this editor supported images or I would have done that yesterday.

**Comment by denisgcm** [comment]
Thank you. Your problem requires deeper investigation. Could you please run this script:

```bash
cd ~
# admin or if you replaced the user, then your gm user name
export GM_USER=admin
export GM_PASSWORD=your_gm_password
export CLUSTER_NAME=your_gm_cluster_name
gmc job list 1>gmd.log 2>&1
gmc cluster logs get-all --name=${CLUSTER_NAME} --save-to=cluster.log  1>>gmd.log 2>&1
```

You will get two files: `cluster.log` and `gmd.log`. Would you please zip the results and send the resulting archive to info@galeracluster.com?

**Comment by dribblecastle** [comment]
The only output I get is to the gmd.log file created per your commands above.   The cluster.log file is not generated.  

Here are the only two lines in the gmd.log file.

```
2022/02/14 12:51:02 remote error: Login failed
2022/02/14 12:51:14 remote error: Login failed
```
I checked my permissions for the root user within MySQL and they are set the same way my working staging setup is configured.

**Comment by denisgcm** [comment]
`GM_USER` and `GM_PASSWORD` are your Galera Manager user and password correspondingly. Hopefully it can help.

**Comment by dribblecastle** [comment]
Thanks.  I was able to figure things out.   I'm pretty sure my issue was due to the change in the logging location.   I changed the location but the correct permissions were not set on the directory containing the log file.  However, I also used the script to do the uninstall and reinstall compared to a **apt-get purge gmd**  that I had done previously in my troubleshooting.   The purge didn't seem to touch the other components used by gmd.
